pkg/main.go

```diff
@@ -13,7 +13,7 @@ import (
 func main() {
 	// Start listening to requests sent from Grafana.
 	s := redshift.New()
-	ds := sqlds.NewDatasource(s)
+	ds := sqlds.NewAsyncDatasource(s)
```

Comment:
This makes it so that the datasource can use an async flow if it's enabled.

> Like if we had an alert, would it have a saved "new async datasource" but would be making a regular datasource query?

I don't think I understand what this means.

Reply:
the async datasource can run sync and async queries. `sqlds` will use the headers to determine what kind of query to run, and if it's an alert query it would have the async datasource and run a regular sync query on it. (im also not sure what the code snippet is)